src/engraving/layout/v0/layoutcontext.h

```diff
@@ -124,6 +176,7 @@ class LayoutContext
 
 private:
     Score* m_score = nullptr;
```

Comment:
What do you think about storing this pointer in DomAccessor only? We can add LayoutContext as a friend class to DomAccessor, and then we can still access m_score without exposing it in the public interface

Reply:
Technically, this would be correct. But semantically, no, because, DOM is the representation of the score as domain model, the context is the representation of the score as its current settings and services. That is, it is semantically not correct to request a score for the context from DOM, because DOM is just a reflection of one of the aspects of this score. 

In the next PR, another structure will appear ... I'll think about how to do it better.